qiskit/providers/models/backendconfiguration.py

```diff
@@ -400,16 +401,41 @@ def acquire(self, qubit: int) -> AcquireChannel:
             raise BackendConfigurationError("Invalid index for {}-qubit systems.".format(qubit))
         return AcquireChannel(qubit)
 
-    def control(self, channel: int) -> ControlChannel:
+    def control(self, qubit: int) -> ControlChannel:
         """
         Return the secondary drive channel for the given qubit -- typically utilized for
         controlling multiqubit interactions. This channel is derived from other channels.
 
         Returns:
             Qubit control channel.
         """
-        # TODO: Determine this from the hamiltonian.
```

Comment:
It doesn't look like this TODO has been completed here

Reply:
Reverted in [0dcf6fd](https://github.com/Qiskit/qiskit-terra/pull/4097/commits/0dcf6fdf599b137341835d2426592b40978f3fa1)